lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -337,8 +345,8 @@ defmodule Kernel.SpecialForms do
 
   The right side of `.` may be a word starting in upcase, which represents
   an alias, a word starting with lowercase or underscore, any valid language
-  operator or any name wrapped in single- or double-quotes. Those are all valid
-  examples:
+  operator, any name wrapped in single- or double-quotes, or be missing. Those
```

Comment:
This special form has two signatures. It is a binary, and also a unary operator, like `-`, but it has no prefix syntax.

Maybe instead of saying that the right operand is missing, we could say that there is only one operand? That wording wanted to be a short edit that in a few informal words gave the idea, but treating this case as a unary operator may be more correct.

In this option we would leave the original text in the "Syntax" section to talk about the right side proper, and then add to the bottom something like:

> When the dot is used to invoke an anonymous function there is only one operand, but it is still written using a postfix notation.

and follow with an example.

This file documents usage and syntax, but it is quite formal because it is about special forms, their representation in the AST, and technical details. In my mind the reader of this documentation is expecting detailed/technical stuff, so this approach seems to square.

What do you think?